metrics/web/test/dashboard/presentation/models/dashboard_page_parameters_model_test.dart

```diff
@@ -0,0 +1,116 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:metrics/dashboard/presentation/models/dashboard_page_parameters_model.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("DashboardPageParametersModel", () {
+    const projectFilter = 'projectFilter';
+    const projectGroupId = 'projectGroupId';
+    const pageParametersMap = {
+      'projectFilter': projectFilter,
+      'projectGroupId': projectGroupId,
+    };
+    const pageParametersModel = DashboardPageParametersModel(
+      projectFilter: projectFilter,
+      projectGroupId: projectGroupId,
+    );
+
+    test(
+      "creates an instance with the given parameters",
+      () {
+        const pageParametersModel = DashboardPageParametersModel(
+          projectFilter: projectFilter,
+          projectGroupId: projectGroupId,
+        );
+
+        expect(pageParametersModel.projectFilter, equals(projectFilter));
+        expect(
+          pageParametersModel.projectGroupId,
+          equals(projectGroupId),
+        );
+      },
+    );
+
+    test(
+      "two instances with the equal fields are equal",
+      () {
+        const firstPageParametersModel = DashboardPageParametersModel(
+          projectFilter: projectFilter,
+          projectGroupId: projectGroupId,
+        );
+        const secondPageParametersModel = DashboardPageParametersModel(
+          projectFilter: projectFilter,
+          projectGroupId: projectGroupId,
+        );
```

Comment:
✅ 
Consider making these variables non-constant as currently, they are not only equal - they are **same**. The [same](https://api.flutter.dev/flutter/package-matcher_matcher/same.html) matcher finishes successfully in this test case. This matcher uses the [identical function](https://api.dart.dev/stable/2.13.4/dart-core/identical.html)) that checks references. The current implementation does not really check the props implementation - it would work without the equatable as well.